kobo/settings/base.py

```diff
@@ -349,6 +349,38 @@
         'having the system empty it automatically.',
         'positive_int_minus_one',
     ),
+    # Enable the use of validators with custom rules
+    'PASSWORD_VALIDATION_CUSTOM_RULES_ENABLED': (
+        True,
+        'Enable custom password validation rules. '
+        'The subsequent rules will not work unless this field is Enabled',
+    ),
+    # Built-in password validator
+    'MINIMUM_LENGTH_VALIDATION': (
+        10,
+        'Minimum length for all passwords',
+    ),
+    # Build-in password validator
+    'USER_ATTRIBUTE_SIMILARITY_VALIDATOR': (
+        True,
+        "Custom user attribute similarity validation is enabled",
+    ),
+    # Built-in password validator
+    'COMMON_PASSWORD_VALIDATION': (
+        '',
+        'Add to list of common passwords for password validation',
```

Comment:
Is it easy to extend the built-in Django list with custom words? It is not required for the task (i.e. we are good with the default list), but it would be nice to have if it's not very time consuming

Reply:
I don't think it will be hard to do at all since we're doing very similar work: https://stackoverflow.com/a/58367983